jpo-ode-core/src/main/java/us/dot/its/jpo/ode/model/OdeTimData.java

```diff
@@ -29,4 +32,16 @@ public OdeTimData(OdeMsgMetadata metadata, OdeMsgPayload payload) {
        super(metadata, payload);
    }
 
+   @Override
+   @JsonTypeInfo(use = Id.CLASS, include = As.EXISTING_PROPERTY, defaultImpl = OdeTimMetadata.class)
+   public void setMetadata(OdeMsgMetadata metadata) {
+      super.setMetadata(metadata);
+   }
+
+   @Override
+   @JsonTypeInfo(use = Id.CLASS, include = As.EXISTING_PROPERTY, defaultImpl = OdeTimPayload.class)
+   public void setPayload(OdeMsgPayload payload) {
+      super.setPayload(payload);
+   }
+
```

Comment:
I added these overrides to the `OdeTimData.java` class to allow for serialization using this class as the parent.

Reply:
Sounds great, thanks!